cytotable/convert.py

```diff
@@ -360,14 +376,16 @@ def _get_table_keyset_pagination_sets(
             page_keys = ddb_reader.execute(sql_query).fetchall()
             page_keys = [key[0] for key in page_keys]
 
-    return _generate_pagesets(page_keys, chunk_size)
+    # The type: mention below is used to ignore a mypy linting error
+    # wherein it considers _generate_pagesets to be invalid.
+    return _generate_pagesets(page_keys, chunk_size)  # type: ignore[return-value]
 
 
 @python_app
 def _source_pageset_to_parquet(
     source_group_name: str,
     source: Dict[str, Any],
-    pageset: Tuple[Union[int, float], Union[int, float]],
+    pageset: Optional[Tuple[Union[int, float], Union[int, float]]],
```

Comment:
I'm not sure this type covers `_get_table_keyset_pagination_sets()` now returning `List[None]`; I believe `Optional[...]` covers just it returning `None`.

Reply:
Thanks for raising this - I think the key here is that this is a "set" and not "sets"; e.g. `[None]` or `[(0,0)]` for "sets" and `None` or `(0,0)` for a "set". I'm going to leave this as-is as a result.